### PR TITLE
Changes in snapcraft.yaml to publish

### PR DESCRIPTION
**Pull request by gabrielcocenza**
- added instrunctions to manually add alias
- removed write access to .local/share/juju
- [juju-client-observe-interface](https://snapcraft.io/docs/juju-client-observe-interface) already has read access to .local/share/juju

### CODE REVIEW

File: snap/snapcraft.yaml [hunk 1]
```diff
@@ -54,11 +52,7 @@ parts:
       craftctl set version=${VERSION}
 
 plugs:
-  dot-local-share-juju:
```
Comment (rgildein) [inline snap/snapcraft.yaml:57]: I think that this is needed, because we are accessing it with lubjuju, or not?

Comment (rgildein) [reply snap/snapcraft.yaml:57]: Simple reading access would be enough, but we need to be able to read from there.

Comment (gabrielcocenza) [reply snap/snapcraft.yaml:57]: The [juju-client-observe-interface](https://snapcraft.io/docs/juju-client-observe-interface) already give us read access to .local/share/juju used in libjuju

Comment (rgildein) [reply snap/snapcraft.yaml:57]: Perfect 👍

File: README.md [hunk 2]
```diff
@@ -11,10 +11,10 @@ different repository, and only use some of the code and ideas found here.
 make clean
 sudo snap remove charmed-openstack-upgrader --purge
 make build
-sudo snap install ./charmed-openstack-upgrader.snap --devmode
+sudo snap install ./charmed-openstack-upgrader.snap --dangerous
 sudo snap connect charmed-openstack-upgrader:juju-client-observe snapd
-sudo snap connect charmed-openstack-upgrader:dot-local-share-juju snapd
-sudo snap connect charmed-openstack-upgrader:local-cou snapd
+sudo snap connect charmed-openstack-upgrader:dot-local-cou snapd
+sudo snap alias charmed-openstack-upgrader.cou cou
```
Comment (agileshaw) [inline README.md:17]: We should state that this is for local builds. It would be easier when we are ready to add `install from snapstore` section

**Review by agileshaw — CHANGES_REQUESTED** [review]

**Review by agileshaw — APPROVED** [review]

**Review by rgildein — APPROVED** [review]

**Pull request merged**